experimental/Schemes/ProjectiveSchemes.jl

```diff
@@ -656,20 +672,25 @@ function fiber_product(f::AbsSpecMor, P::ProjectiveScheme{<:MPolyQuoLocalizedRin
                                )
 end
 
-fiber_product(X::AbsSpec, P::ProjectiveScheme{<:MPolyQuoLocalizedRing}) = fiber_product(inclusion_map(X, base_scheme(P)), P)
+fiber_product(X::AbsSpec, 
+              P::ProjectiveScheme{<:Union{<:MPolyRing, <:MPolyQuo, <:MPolyLocalizedRing, <:MPolyQuoLocalizedRing}}
+             ) = fiber_product(inclusion_morphism(X, base_scheme(P)), P)
 
 ### canonical map constructors
 
 @Markdown.doc """
-    inclusion_map(P::T, Q::T)
+    inclusion_morphism(P::T, Q::T)
 
 Assuming that ``P ⊂ Q`` is a subscheme, both proper over an inclusion of 
 their base schemes, this returns the associated `ProjectiveSchemeMor`.
```

Comment:
```suggestion
Assuming that ``P ⊂ Q`` is a subscheme, both proper over their base schemes, 
this returns the associated `ProjectiveSchemeMor`.
```
Not sure what over an inclusion of their base schemes means. Do you mean the structure morphism? But that is to the base scheme.

Reply:
This is just the fiber product over Y-> X and IP^r_X ->X with Y a proper subset of X.